pkg/rtc/transportmanager.go

```diff
@@ -20,6 +21,12 @@ import (
 
 const (
 	failureCountThreshold = 2
+
+	// when RR report loss frac over this threshold, we consider it is a unstable event
```

Comment:
maybe modify comment here to say that this percentage (25%). There is conversion below to RR fraction lost scale where 25% will become 64 or so. As RTCP calls it loss fraction, suggesting comment here as percentage.